zip-draft-XXXX.rst

```diff
@@ -0,0 +1,298 @@
+::
+
+  ZIP: Unassigned
+  Title: Lockbox for Decentralized Grants Allocation
+  Owners: Skylar Saveland <skylar@free2z.com>
+          Jason McGee <>
+  Credits: @daira
+           @str4d
+           @nuttycom
+  Status: Draft
+  Category: Consensus
+  Created: 2024-06-26
+  License: MIT
+  Pull-Request: <https://github.com/zcash/zips/pull/858>
+
+
+Terminology
+===========
+
+The key words "MUST", "REQUIRED", "MUST NOT", "SHOULD", and "MAY" in this
```

Comment:
TODO: do a pass for when these MUST be capitalized ;9

Reply:
I'm pretty happy with it, I think. Some things we might want to look at:

1. replace the comment about terminology with any actual terminology we should define
2. Capitalize "MUST", "REQUIRED", "MUST NOT", "SHOULD", and "MAY" where it makes sense
3. Improve the references (?)

In addition, the `Specification`  section might need some more revisions? Does it need to be more specific and more technical or is it good enough?

Reply:
Maybe in `non-requirements` it could be mentioned that this proposal could sit alongside other things like "ZSF" and some form of continuation of the "direct streams" current funding model (eg some percentage to ZCG?). I think the "lockbox" ZIP as written doesn't preclude other things that could be done with the block reward?

Reply:
I think another primary motivation is that the lockbox serves an important purpose if the community cannot come to consensus on a dev fund proposal or if there is no support for a direct funding model. By allocating 20% of the block rewards to the lockbox, the dev fund can continue to accrue. Without this, the 20% would go to miners, and redirecting these rewards from miners to a new Dev Fund later would require a future network upgrade, which could be controversial.

Reply:
> Maybe in `non-requirements` it could be mentioned that this proposal could sit alongside other things like "ZSF" and some form of continuation of the "direct streams" current funding model (eg some percentage to ZCG?). I think this "lockbox" ZIP as written doesn't preclude other things that could be done with the block reward?

Re the ZSF: It's worth noting that the lockbox and ZSF are compatible. As Daira points out in hir writeup, the deferred funds held in the lockbox are part of circulating supply, whereas the ZSF contains unissued supply. Ze says, "mixing these two is a bad idea." Also, the primary (or sole) use case for the lockbox is to temporarily serve as a reserve for the dev fund until the community decides how it should be allocated. In contrast, the ZSF has a number of different long-term use cases.

Re: "some percentage to ZCG": Yes, it makes sense to reference a ZCG allocation ini the non-requirements section. I mentioned this to strad in the ZIP Editor office hours and said for a hybrid version where a portion goes to the lockbox and a portion to ZCG, we can use ZIP 207 as a template to define the funding stream to ZCG.  I'll take a look at that later today and will follow up with you.

Reply:
Just to clarify, the write-up was written jointly by me, @str4d, and @nuttycom .